Packs/Hackuity/ReleaseNotes/1_0_7.md

```diff
@@ -1,3 +1,3 @@
 #### Integrations
 ##### Hackuity
-- Updated the Docker image to: *demisto/python3:3.10.11.61265*.
+- Updated the Docker image to: *demisto/python3:3.10.11.61265*.
```

Comment:
Why do we see this change? Please revert (maybe a new line at the end of the file is missing)

Reply:
Done, indeed a new line in the end of file.